FWCore/Services/scripts/edmTracerCompactLogViewer.py

```diff
@@ -543,7 +906,24 @@ def textOutput( parser ):
     parser.add_argument('-f', '--frameworkOnly',
                         action='store_true',
                         help='''Output only the framework transitions, excluding the individual module transitions.''')
+    parser.add_argument('-j', '--json',
+                        action='store_true',
+                        help='''Write output in json format.''' )
+    parser.add_argument('-w', '--web',
+                        action='store_true',
+                        help='''Writes data.js file that can be used with the web based inspector. To use, copy directory ${CMSSW_RELEASE_BASE}/src/FWCore/Services/template/web to a web accessible area and move data.js into that directory.''')
```

Comment:
How about adding a `README.md` that describes an example how to use the `Tracer` service and this script to create the `data.js` and use the web viewer?